### DISCUSSION TITLE
Question: Command Bar Borders - Are these correct?

### DISCUSSION THREAD

**Discussion by mdtauk** [microsoft/microsoft-ui-xaml#7463] [Q&A]
The borders which appear around the edge of the CommandBars as they expand/pop-over - should they appear so bright and solid?
![image](https://user-images.githubusercontent.com/7389110/122811119-fe32d080-d2c7-11eb-9ac3-b83c7ce1a252.png)
![image](https://user-images.githubusercontent.com/7389110/122811202-1571be00-d2c8-11eb-8a95-444cf4d4e452.png)

When comparing to the similar CommandBarFlyout controls - the borders for the CommandBars look garish

![image](https://user-images.githubusercontent.com/7389110/122810751-9e3c2a00-d2c7-11eb-84a3-5b723439e80e.png)
![image](https://user-images.githubusercontent.com/7389110/122810798-ab591900-d2c7-11eb-9927-86ef55ee8869.png)
_CommandBarFlyouts use a darker Acrylic and don't feature prominent borders_

I am sure they look more subtle when used on the default Light or Dark themed backgrounds, but with the Rainbow background on the testing page, but maybe something with lower opacity may be better, like the **CardStrokeColorDefault**

![image](https://user-images.githubusercontent.com/7389110/122812279-6635e680-d2c9-11eb-997f-0a275faead7c.png)
![image](https://user-images.githubusercontent.com/7389110/122812696-ea886980-d2c9-11eb-9f03-778c6ad87242.png)
_Examples on solid "expected" backgrounds_

**Comment by StephenLPeters** [comment]
@chigy thoughts?
